src/components/FilterPanel.jsx

```diff
@@ -53,7 +53,7 @@ export default function FilterPanel({ onClose }) {
   }, [location]);
 
   const filters = {
-    'What are you studying?': {
+    whatAreYouStudying: {
```

Comment:
Gotta do something like this for `yarn i18next` to pick this up as a translation key:
```suggestion
    [t('whatAreYouStudying')]: {
```

Here and elsewhere.